src/components/Page/Toolbar/Toolbar.css

```
@@ -15,6 +15,14 @@
   padding: 0px;
 }
 
+.toolbar div.left {
```

Ooh, I have an even better idea.
```suggestion
.top-toolbar.left {
```